src/index.html

```diff
@@ -15,6 +15,5 @@
   
         </ul>    
       </section>
-     
     </body>
```

Comment:
Not yet solved

Reply:
Look at the image below, the first and last tasks seem to be completed and mark as true however, it is not completed because I didn't mark the checkbox as completed.

![rr10](https://user-images.githubusercontent.com/55635977/134535669-a4c11acb-01ac-4a22-8a0b-033772ca7b40.png)

Reply:
Great job now
![rr11](https://user-images.githubusercontent.com/55635977/134536533-5dff6c5e-4f3f-450f-b4bb-b378a00b6dd2.png)

Reply:
Solve eslint errors

Reply:
Good Job